eng/common/scripts/Helpers/ApiView-Helpers.ps1

```
@@ -238,3 +238,44 @@ function Save-PackageProperties ($repoRoot, $serviceNames, $outputDirectory) {
     & $scriptPath -ServiceDirectory $serviceDirectory.Trim() -OutDirectory $outputDirectory
   }
 }
+
+function Create-API-Review {
+  param (
+    [string]$apiviewEndpoint = "https://apiview.dev/PullRequest/DetectAPIChanges",
+    [string]$specGenSDKArtifactPath,
+    [string]$apiviewArtifactName,
+    [string]$buildId,
+    [string]$commitish,
+    [string]$repoName,
+    [string]$pullRequestNumber
+  )
+  . ${PSScriptRoot}\..\logging.ps1
+  $specGenSDKContent = Get-Content -Path $SpecGenSDKArtifactPath -Raw | ConvertFrom-Json
+  $uri = [System.UriBuilder]$apiviewEndpoint
+  $query = [System.Web.HttpUtility]::ParseQueryString('')
+  $query.Add('artifactName', $apiviewArtifactName)
+  $query.Add('buildId', $buildId)
+  $query.Add('commitSha', $commitish)
+  $query.Add('repoName', $repoName)
+  $query.Add('pullRequestNumber', $pullRequestNumber)
+  
+  foreach ($requestData in $specGenSDKContent.apiViewRequestData) {
+    $requestUri = $uri
+    $requestParam = $query
+    $requestParam.Add('packageName', $requestData.packageName)
+    $requestParam.Add('filePath', $requestData.filePath)
+    $requestParam.Add('language', $requestData.language)
+    $requestUri.query = $requestParam.toString()
+    LogInfo "Request URI: $($requestUri.Uri.OriginalString)"
+
+    try
+    {
+      $response = Invoke-WebRequest -Method 'GET' -Uri $uri.Uri -MaximumRetryCount 3
```

We will need to report the errors and details for based on response codes.